src/TestTask/components/CalculatorSwitch/CalculatorSwitch.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Tab, Tabs, useMediaQuery } from '@material-ui/core'
+import { theme } from '@static/theme'
+import { useSingleTabStyles, useTabsStyles } from './style'
+
+export interface IProps {
+  onSwitch: (isConcentrated: boolean) => void
+  className?: string
+  style?: React.CSSProperties
+  currentCurrency: number
+  setCurrentCurrency: (currentCurrency: number) => void
+}
+
+export const CalculatorSwitch: React.FC<IProps> = ({
+  onSwitch,
+  className,
+  style,
+  currentCurrency,
+  setCurrentCurrency
+}) => {
+  const isXs = useMediaQuery(theme.breakpoints.down('xs'))
+
+  const tabsClasses = useTabsStyles({ value: currentCurrency })
+  const singleTabClasses = useSingleTabStyles()
+
+  const handleChange = (_: React.ChangeEvent<{}>, newValue: number) => {
+    setCurrentCurrency(newValue)
+    onSwitch(!newValue)
```

Comment:
there are three values in this switch, so this line will not work properly here; I know that you've copied it from other component, but that was other case, where there are only two values

Reply:
Thanks.